src/avram/save_operation.cr

```diff
@@ -153,6 +153,11 @@ abstract class Avram::SaveOperation(T) < Avram::Operation
       end
 
       def set_{{ attribute[:name] }}_from_param(_value)
+        {% if attribute[:nilable] %}
+          # The type is nilable, but if the value is blank, it will return a FailedCast
+          # in types like `Time?` or `Int32?`, etc...
+          return nil if _value.blank?
```

Comment:
Actually I think we should change this slightly. If `_value.blank?` then I think it should always return `nil` and never attempt parsing. Otherwise when you send an empty string for a `Time` it will have both a "is missing" and a "is invalid". 

If we always return `nil` then it would just have "is required" which is the expected behavior IMO

Reply:
oh, interesting point. Yeah, I think that will work. I'll try that

Reply:
Actually... Doing that causes 2 of the specs to fail. This kind of makes me think there may be a behavior change that's not expected... Think I should just fix those specs? or just roll with what I have now?

```
1) Avram::SaveOperation .update on failure yields the operation and nil
     Failure/Error: operation.save_failed?.should be_true

       Expected: true
            got: false

     # spec/save_operation_spec.cr:531

  2) Avram::SaveOperation .update! on failure raises an exception
     Failure/Error: expect_raises Avram::InvalidOperationError do

       Expected Avram::InvalidOperationError but nothing was raised

     # spec/save_operation_spec.cr:584
```